source/features/clean-repo-sidebar.tsx

```diff
@@ -58,9 +67,10 @@ async function moveReportLink(): Promise<void> {
 	await domLoaded;
 
 	const reportLink = $('.Layout-sidebar a[href^="/contact/report-content"]')?.parentElement;
-	if (reportLink) {
-		// Your own repos don't include this link
-		$('.Layout-sidebar .BorderGrid-row:last-of-type .BorderGrid-cell')!.append(reportLink);
+	if (reportLink) { // Your own repos don't include this link
+		setTimeout(() => { // Wait for other hide init methods to run first before checking which is not hidden
```

Comment:
If we need to wait for it, then maybe it should be called by it, no need for arbitrary timeouts